core/templates/dev/head/player/flag_exploration_popup.html

```diff
@@ -0,0 +1,47 @@
+<script type="text/ng-template" id="modals/flagExploration" ng-controller="flagExplorationPopup">
+  <div class="modal-header">
+    <h3>
+      <span>Report Exploration</span>
+    </h3>
+  </div>
+  <div class="modal-body oppia-long-text">
+    <form>
+      <div class="radio">
+      <label><input type="radio" name="AD" value="AD" ng-model="flag" ng-change="triggerOther(flag)">It is an Ad</label>
```

Comment:
Also this line is contained within the previous div, so it should be indented by 2 from it.